code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1020,6 +1020,18 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	dat += "<br><a href='?src=[REF(src)];[HrefToken()];add_station_goal=1'>Add New Goal</a>"
 	usr << browse(dat, "window=goals;size=400x400")
 
+proc/immerse_player(mob/living/carbon/target, toggle=TRUE, remove=FALSE)

```

Comment:
Is this the right time to announce how much I hate that a lot of admin procs are global? Admin datum when?

Reply:
We have an admin datum

Reply:
when you code it